packages/mermaid/src/setupGraphViewbox.js

```diff
@@ -25,6 +25,7 @@ export const calculateSvgSizeAttrs = function (height, width, useMaxWidth) {
   if (useMaxWidth) {
     attrs.set('width', '100%');
     attrs.set('style', `max-width: ${width}px;`);
+    // TODO: when using max width it does not set height? Is it intended?
```

Comment:
Not 100% sure, but maybe the height scales with the width?

Reply:
Have not figured it out yet. Seems that this diagram layout calculates real nodes dimensions based on overall height proportionally. Not specifying height at my laptop makes this diagram half-cropped